server.py

```diff
@@ -18,13 +21,18 @@ async def create_url(request: Request):
             insert(DATABASE, urljson['url'], urljson['alias'])
             print("inserted successfully")
             return {"url":urljson['url'], "alias":urljson['alias']}
+        if args.disable_random_alias:
+            raise HTTPException(status_code=400, detail="Random alias generation is disabled")
```

Comment:
raise a KeyError instead, when random gen is disabled we require it in the request. add a corresponding catch for KeyError that raises an HTTPException there